<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddingMode</class>
 <widget class="QMainWindow" name="AddingMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>512</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>440</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>221</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>100</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>131</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>171</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>91</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&gt;&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&gt;&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="addToTimeSlotButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Me To This 
Time Slot</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>91</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&gt;&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&gt;&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Click on an event to see its 
scheduled time slots.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Events</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Time Slots</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Attendees</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Click on a time slot to see 
who is attending.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Click on the &quot;Add Me To 
Time Slot&quot; to add yourself 
to event.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
